common/scripts/lib/_parseArgs.sh

```diff
@@ -85,6 +85,7 @@ __validate_runtime() {
   ################## check db url ##################################
   if [ "$DB_IP" == "" ]; then
     __process_msg "DB_IP is not set"
+    __set_install_db
```

Comment:
this means that installer is asking more questions that it should for a completely onebox installation. the flow should be something like this
- ask the IP address of db (type D to use defautl)
    - if default is chosen, move to next step
- if some IP is provided, __then__ ask if a new installation is required or an existing one should be used
   - for new installation, run scripts
   - for existing ones, connect